presentation.ipynb

```diff
@@ -122,7 +122,8 @@
     " - `ndarray`: A generic, n-dimensional array data structure\n",
     " - Sophisticated machinery for operating on array data (vectorization, broadcasting, `ufuncs`)\n",
     " - Language extension/integration (C-API, `f2py`)\n",
-    "   * [Array API](https://docs.scipy.org/doc/numpy/reference/c-api.array.html) for accessing/extending array functionality in external libraries"
+    "   * [Array API](https://numpy.org/doc/1.17/reference/c-api.array.html) for accessing/extending array functionality in external libraries\n",
+    " - Protocols for mimicing the very successful NumPy Python interfaces for other easey interoperability with other packages (more later)"
```

Comment:
Not sure if this needs a talking point here, feel free to remove

Reply:
Thanks, nice catch

Reply:
Thanks, nice catch

Reply:
Regarding the added bullet-point about protocols, did you have the `__array_function__` and `__array_module__` protocols in mind, or something less specific. I just want to make sure I fully understand the suggestion

Reply:
I was trying to catch the essence of all the efforts we have made over the years. The latest are NEP 37, `__array_function__` and `__array_ufunc__`. There is also `__array__` and `__array_wrap__` for subclassing `ndarray`. Then  the [PEP 3118](https://www.python.org/dev/peps/pep-3118/) buffer protocol, by Travis Oliphant, that solves the interchange problem for non-object arrays and it turns out we have an earlier [Array Interface](https://numpy.org/devdocs/reference/arrays.interface.html). I might have missed one or two more.